aspnet/fundamentals/request-features.rst

```diff
@@ -12,63 +12,61 @@ Individual web server features related to how HTTP requests and responses are ha
 Feature interfaces
 ------------------
 
-ASP.NET Core defines a number of `HTTP feature interfaces <https://docs.asp.net/projects/api/en/latest/autoapi/Microsoft/AspNetCore/Http/Features/index.html>`_, which are used by servers to identify which features they support. The most basic features of a web server are the ability to handle requests and return responses, as defined by the following feature interfaces:
+ASP.NET Core defines a number of HTTP feature interfaces in the :dn:ns:`Microsoft.AspNetCore.Http.Features` namespace, which are used by servers to identify which features they support. The most basic features of a web server are the ability to handle requests and return responses, as defined by the following feature interfaces:
```

Comment:
>which are used by servers to identify which features they support

which identify the features they support

I'd write it like this, altho it might be too terse for your taste -

+ASP.NET Core defines a number of HTTP feature interfaces in :dn:ns:`Microsoft.AspNetCore.Http.Features` which identify the features they support. The following feature interfaces handle requests and return responses:

Those are not the most basic features, altho they are probably the most fundamental. But we don't need to point out the obvious.

Your version is fine, just now how I'd write it. Mine is not vastly improved, but it is less wordy.